library/core/src/str/validations.rs

```diff
@@ -229,31 +245,31 @@ pub(super) fn run_utf8_validation(v: &[u8]) -> Result<(), Utf8Error> {
     Ok(())
 }
 
-// https://tools.ietf.org/html/rfc3629
```

Comment:
this change seems unnecessary, please undo it.

Reply:
I can undo moving this item into the function and move it back here, but `static`->`const` change is required since `const fn` can't refer to statics...